Sources/GDSCommon/Patterns/GDSInformation/GDSInformationViewModel.swift

```diff
@@ -20,20 +20,50 @@ import UIKit
 /// screens modally and calling custom methods when screens appear and dismiss.
 /// For example, this might include tracking an analytics screen view, but it could be used
 /// for other code such as making an API call.
+
+@available(*, deprecated, renamed: "GDSInformationViewModelV2", message: "Should conform to additional protocols if additional configuration is required")
+public typealias GDSInformationViewModel = GDSInformationViewModelV2 &
+                                           GDSInformationViewModelOptionalFootnote &
+                                           GDSInformationViewModelPrimaryButton &
+                                           GDSInformationViewModelOptionalSecondaryButton
+
 @MainActor
-public protocol GDSInformationViewModel {
+public protocol GDSInformationViewModelV2 {
     var image: String { get }
     var imageWeight: UIFont.Weight? { get }
     var imageColour: UIColor? { get }
     var imageHeightConstraint: CGFloat? { get }
     var title: GDSLocalisedString { get }
     var body: GDSLocalisedString? { get }
+}
+
+@MainActor
+public protocol GDSInformationViewModelOptionalFootnote {
     var footnote: GDSLocalisedString? { get }
+}
+
+@MainActor
+public protocol GDSInformationViewModelPrimaryButton {
     var primaryButtonViewModel: ButtonViewModel { get }
+}
+
+public extension GDSInformationViewModelPrimaryButton {
+    var primaryButtonViewModel: ButtonViewModel {
+        FallBackButtonViewModel.primary
+    }
+}
+
+@MainActor
+public protocol GDSInformationViewModelOptionalPrimaryButton {
+    var primaryButtonViewModel: ButtonViewModel? { get }
+}
+
+@MainActor
+public protocol GDSInformationViewModelOptionalSecondaryButton {
     var secondaryButtonViewModel: ButtonViewModel? { get }
 }
 
 @MainActor
-public protocol GDSInformationViewModelWithChildView {
```

Comment:
The previous name made more sense to me.. because the view model _has_ (rather than _is_) a child view, but happy to discuss 🤷🏼